### DISCUSSION TITLE
从v0.4.1升级到v0.6.1发现语音循环播放有问题

### DISCUSSION THREAD

**Discussion by xrz-baizer** [ZuodaoTech/everyone-can-use-english#1180] [Q&A]
在v0.6.1版本中截取一段音频，在**播放单句**模式下，重复播放几次后会自动播放下一句。
后回退至v0.5版本也有该问题，逐回退至0.4.1版本才恢复正常。
<img width="1280" alt="image" src="https://github.com/user-attachments/assets/1552361f-289d-4006-9965-25b117b3a5b9">
另外，右侧展示区中的句子希望可以增加一个调节字体大小的功能，有时候在高分辨的显示器下会显示的有点小。

<img width="268" alt="image" src="https://github.com/user-attachments/assets/5dc84f1c-15b4-46db-86ce-4bbfff5c9060">

**Comment by an-lee** [comment]
循环播放好像是不少反馈有问题，我再优化下。

可以先手动播放吧，就按一下空格。

**Reply by xrz-baizer** [reply]
使用旧版本的时候，为什么是强制更新的呀，新版版本还是没有修复这个问题

**Comment by an-lee** [comment]
能把会出现这个问题的音频发一下吗？具体是哪一段，我看能不能重现出来，然后看看啥问题。

**Reply by xrz-baizer** [reply]
应该任意一段音频都有这个问题。
我是在某个句子中选取了一段（so that it would be restful）进行反复播放，播放几次后就会超出这个范围，播放下一句去了
<img width="1155" alt="image" src="https://github.com/user-attachments/assets/dda03686-5949-40e3-aded-22fa4380aca6">

**Reply by xrz-baizer** [reply]
不怎么好复现，最新版072确实有个问题
<img width="1091" alt="image" src="https://github.com/user-attachments/assets/008bb590-cd0b-4f2c-947a-444e042eccb3">

**Reply by an-lee** [reply]
应该是某些音频某些部分才会出现这个问题。你把能稳定复现这个问题的音频，下载下来，打包 zip 文件上传到这里来看看。

**Reply by xrz-baizer** [reply]
[Lucy1dialogslow.mp3.zip](https://github.com/user-attachments/files/17879995/Lucy1dialogslow.mp3.zip)
又试了一下，在（so that it would be restful）接近结束前按空格暂停，再按空格就有概率复现。

不会稳定复现，我其它音频好像是正常的

**Reply by an-lee** [reply]
好的，我来测试一下。
